fdmi/plugins/sched.c

```diff
@@ -509,60 +499,46 @@ static int sched_init(struct sched_conf *conf)
 	dix_conf.kc_create_meta 	 = false;
 	m0_conf.mc_idx_service_conf 	 = &dix_conf;
 
-	write_plugin_endpoint(conf->local_addr);
-
 	/* Client instance */
 	rc = m0_client_init(&m0c, &m0_conf, true);
-	if (rc != 0) {
-		fprintf(stderr, "Failed to initialise Client: %d\n", rc);
-		goto do_exit;
-	}
+	if (rc != 0)
+		return rc;
 
 	M0_POST(m0c != NULL);
 
 	rc = fdmi_service_start(m0c);
```

Comment:
@yumanets  @huanghua78 @max-seagate @nikitadanilov  Can you check if below suggestion is feasible for motr client api?

Something to think for near future in ease fdmi support. If we can have `m0_fdmiapp_init` or `m0_fdmi_client_init` exposed from motr client that can internally do both `m0_client_init` and what is done inside `fdmi_service_start()` here. 

This way fdmi app developers will have a single api to call and they don't have to repeat code inside `fdmi_service_start` ?

Reply:
This is a good idea if you ask me.